### PR TITLE
Add version v0.10.1 of triggers

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Add version v0.10.1 of triggers to docs

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: sync/config/triggers.yaml [hunk 1]
```diff
@@ -30,6 +30,34 @@ archive: https://github.com/tektoncd/triggers/tags
...
+  # Dict of folders to sync
+  folders:
+    docs:
+      index: README.md
+      include: ['*.md']
+    docs/getting-started:
+      target: getting-started
+      include:
+      - create-ingress.yaml
+      - create-webhook.yaml
```
Comment (AlanGreene) [inline sync/config/triggers.yaml:60]: looks like 0.9.1 is already included

Comment (afrittoli) [reply sync/config/triggers.yaml]: heh, thanks! updated

**Review by AlanGreene — APPROVED** [review]
Thanks @afrittoli

**Comment by AlanGreene** [comment]
Looks like the netlify build failed on `pip install`, closing and reopening the PR to re-trigger the checks.
/close

**Comment by AlanGreene** [comment]
That didn't work... if someone has access to the netlify account they may be able to manually re-trigger the checks, otherwise looks like this PR needs a new commit.

**Comment by afrittoli** [comment]
Seriously GitHub? https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks#skipping-and-requesting-checks-for-individual-commits

**Comment by afrittoli** [comment]
The integration with Netlify is via GitHub App, installed in the tektoncd org. 
I checked the configuration of the app and it has no settings, it is not possible to select which events trigger the CI check.
I checked GitHub docs, and they suggest to [do a commit](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks#skipping-and-requesting-checks-for-individual-commits) to retrigger the GitHub app, which can work but is a bit... meh.
One thing that I could confirm is that if I re-run the deploy via the Netlify website, all CI jobs are updated.
I think we need to build a small bot that reacts to a github comment and re-triggers the build using an API token. It might even be possible via GitHub actions.

**Pull request merged**